### PR TITLE
Use static tag for resources in versions, add  for partials

### PR DESCRIPTION
**Pull request by joshblum**
WIP

### CODE REVIEW

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -126,3 +126,10 @@ For stylesheets we only compile scss files so if your file is at
 ``orchestra/common/static/common/scss/example.scss``, to include it in an HTML
 file you should write ``{% static 'common/css/example.css' %}">`` as the static
 file path.
+
```
Comment (marcua) [inline docs/source/contributing.rst:129]: any other .rst updates in docs necessary?

File: orchestra/templates/orchestra/base.html [hunk 2]
```diff
@@ -75,6 +75,16 @@
     {% block main %}
     {% endblock %}
 
+    {% block static_url %}
+      <script type="text/javascript">
+        // NOTE(joshblum): This is a noop function that is replaced by django static file
+        // processing, but allows us to work in dev.
+        function $static(url) {
```
Comment (marcua) [inline orchestra/templates/orchestra/base.html:82]: my javascript-foo isn't strong enough to know, but should this be:

`window.$static = function(url) {...`

File: orchestra/templates/orchestra/index.html [hunk 3]
```diff
@@ -93,12 +93,12 @@
 <script src="{% static "orchestra/project_management/js/projectVis.js" %}" type="text/javascript"></script>
 
 {% for js in javascript_includes %}
-    <script src="{{ js }}" type="text/javascript"></script>
+  {{ js | safe }}
```
Comment (marcua) [inline orchestra/templates/orchestra/index.html:97]: no `{% static...`?

File: orchestra/templates/orchestra/index.html [hunk 4]
```diff
@@ -93,12 +93,12 @@
...
 {% endfor %}
 {% endblock page_javascript%}
 
 {% block page_stylesheets %}
     {% for css in stylesheet_includes %}
-        <link href="{{ css }}" rel="stylesheet">
+      {{ css | safe }}
```
Comment (marcua) [inline orchestra/templates/orchestra/index.html:104]: same?

File: orchestra/views.py [hunk 5]
```diff
@@ -43,6 +44,17 @@
 UserModel = get_user_model()
 
 
+# NOTE(joshblum): whitenoise is a bit over eager and tries to replace our
```
Comment (marcua) [inline orchestra/views.py]: i don't quite get this---explain here or offline?

File: orchestra/templates/orchestra/index.html (hunk 3 above)
Comment (marcua) [inline orchestra/templates/orchestra/index.html:97]: oh i see---should we have a comment here saying to check out `orchestra/views.py` for why we lazily use `{% static`

File: docs/source/contributing.rst (hunk 1 above)
Comment (joshblum) [inline docs/source/contributing.rst:129]: dont think so, can you think of any?

File: orchestra/templates/orchestra/base.html (hunk 2 above)
Comment (joshblum) [inline orchestra/templates/orchestra/base.html:82]: @derek-schultz what do you think?

File: orchestra/templates/orchestra/index.html (hunk 3 above)
Comment (joshblum) [inline orchestra/templates/orchestra/index.html:97]: kk

File: orchestra/templates/orchestra/index.html (hunk 4 above)
Comment (joshblum) [inline orchestra/templates/orchestra/index.html:104]: yup :)

File: orchestra/templates/orchestra/base.html (hunk 2 above)
Comment (joshblum) [inline orchestra/templates/orchestra/base.html:82]: @derek-schultz says "they're equivalent"

**Pull request merged**